addons/core/stringtable.xml

```diff
@@ -347,10 +371,16 @@
             <Key ID="STR_TFAR_SETTINGS_intercomDucking">
                 <Original>Intercom Ducking</Original>
                 <Japanese>インターコムを絞る</Japanese>
+                <English>Intercom Ducking</English>
+                <Chinese>對講機自動迴避音量</Chinese>
+                <Chinesesimp>对讲机自动回避音量</Chinesesimp>
             </Key>
             <Key ID="STR_TFAR_SETTINGS_intercomDucking_desc">
                 <Original>Intercom volume reduction on incoming vehicle radio transmission</Original>
                 <Japanese>車載無線機で受信中はインターコムの音量を絞ります</Japanese>
+                <English>Intercom volume reduction on incoming vehicle radio transmission</English>
```

Comment:
English can be left away, game will default to Original. And that way we don't have duplicates for every string